shared/profile/wot-author/index.stories.tsx

```diff
@@ -1,17 +1,284 @@
 import * as React from 'react'
 import * as Sb from '../../stories/storybook'
-import {Question1, Question2} from '.'
+import {Question1, Question2, Proof} from '.'
+import sortBy from 'lodash/sortBy'
+
+// snapshot of tracker2 assertions: DEBUGStore.getState().tracker2.usernameToDetails.get("mlsteele").assertions
```

Comment:
Should this be here?

Reply:
Yes, it's describing what the large sampleAssertions variable is and how to update it. In case a different example is desired or the format changes. Would more words make that more clear?